airbyte-config/config-persistence/src/main/java/io/airbyte/config/persistence/DatabaseConfigPersistence.java

```diff
@@ -1917,7 +1917,12 @@ static boolean hasNewVersion(final String currentVersion, final String latestVer
 
   @VisibleForTesting
   static boolean hasNewPatchVersion(final String currentVersion, final String latestVersion) {
-    return new AirbyteVersion(latestVersion).checkOnlyPatchVersionIsUpdatedComparedTo(new AirbyteVersion(currentVersion));
+    try {
```

Comment:
for my understanding, why does this have to change to load configs from the remote?

Reply:
It doesn't - it's an unrelated error that has been around that I fixed while I was in here. Basically if you had anything in your database that didn't follow semver (e.g. a `dev` version of a connector) the bootloader would crash when updating definitions

Reply:
got it. I think you pointed it out in your PR description - will also be nice to leave a comment here for future PRs so it's easy to associate :)